### DISCUSSION TITLE
CI for Testing Build

### DISCUSSION THREAD

**Discussion by mitsu00** [ROM-builders/temporary#21647] [General]
Hi sir, I made this CI repo for testing build before they do PR, so this can minimize failed builds on your persistence worker. This is using free Community Container that provided by cirrus-ci. I hope everyone can use this before do pull request.

This CI do:
```
mka sepolicy
mka bootimage
mka init
```
(It's customizable)

Repository: https://github.com/mitsu00/ci-tester
Build result: https://cirrus-ci.com/task/6271775495946240

**Comment by Apon77** [comment]
Don't use free tier of cirrus to do so. Use other option if have. You may get blocked by cirrus.

**Reply by mitsu00** [reply]
If that so, maybe GitHub action is might another option, but they have a limited resources. But let me try. Thanks for the answer 🙏
